build-common/NHibernate.props

```diff
@@ -24,7 +24,7 @@
     <PackageProjectUrl>http://nhibernate.info</PackageProjectUrl>
     <PackageIconUrl>https://raw.githubusercontent.com/nhibernate/nhibernate-core/master/logo/NHibernate-NuGet.png</PackageIconUrl>
     <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
-    <PackageLicenseUrl>https://raw.githubusercontent.com/nhibernate/nhibernate-core/master/LICENSE.txt</PackageLicenseUrl>
+    <PackageLicenseFile>NHibernate.license.txt</PackageLicenseFile>
```

Comment:
Can't we use [`PackageLicenseExpression`](https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/nuget#important-nuget-package-metadata) with value `LGPL-2.1-only` instead? This should allow Nuget to directly states the package license, instead of having to read the file.

Reply:
Done. `LGPL-2.1-only` it is.